client/core/core.go

```diff
@@ -391,10 +393,11 @@ func (c *Core) Exchanges() map[string]*Exchange {
 	infos := make(map[string]*Exchange, len(c.conns))
 	for uri, dc := range c.conns {
 		infos[uri] = &Exchange{
-			URL:        uri,
-			Markets:    dc.markets(),
-			Assets:     dc.assets,
-			FeePending: dc.acct.feePending(),
```

Comment:
@buck54321 with these changes, the method `(*dexAccount).feePending()` is not being used elsewhere. Should we just remove it?

Reply:
Yes, please.